sympy/physics/vector/point.py

```diff
@@ -471,7 +471,7 @@ def v2pt_theory(self, otherpoint, outframe, fixedframe):
         self.set_vel(outframe, v + (omega ^ dist))
         return self.vel(outframe)
 
-    def vel(self, frame):
+    def vel(self, frame, otherpoint = None):
```

Comment:
PEP8 and keyqord-only args would be better, as
```suggestion
    def vel(self, frame, *, other_point=None):
```